docs/notes.md

```diff
@@ -603,4 +603,254 @@ sends a pair \\((L\_j,R\_j)\\) of points at each step \\(j = k\dots1\\). An
+
+\\[
+\begin{aligned}
+  t(x) &= \sum_{j=0}^{m-1} t\_{(j)}(x)\\\\
+  {\mathbf{l}}(x) &= {\mathbf{l}}\_{(0)}(x) || {\mathbf{l}}\_{(1)}(x) || \dots || {\mathbf{l}}\_{(m-1)}(x) \\\\
+  {\mathbf{r}}(x) &= {\mathbf{r}}\_{(0)}(x) || {\mathbf{r}}\_{(1)}(x) || \dots || {\mathbf{r}}\_{(m-1)}(x) \\\\
+\end{aligned}
+\\]
+
+We can add the \\(t_{(j)}(x)\\) values together to create \\(t(x)\\) instead of taking a linear combination of \\(t_{(j)}(x)\\) values, because each \\(t_{(j)}(x)\\) is calculated with the \\(\mathbf{y}^n\_{(j)}\\) and \\(z_{(j)}\\) challenge variables that are unique to that party`j`, so all of the \\(t_{(j)}(x)\\) values will be offset from one another.
+
+Now instead of having to do `m` individual checks to prove that \\(t_{(j)}(x)\\), \\({\mathbf{l}}\_{(j)}(x)\\), and \\({\mathbf{r}}\_{(j)}(x)\\) for all parties `j` are correct, we can do the verification with one check:
+
+\\[
+\begin{aligned}
+  t(x) \stackrel{?}{=} {\langle {\mathbf{l}}(x), {\mathbf{r}}(x) \rangle}
+\end{aligned}
+\\]
+
+We can do this check using the [inner product proof](index.html#inner-product-proof), in the same way the single-value range proof uses the inner product proof.
+
+Proving that \\(t_0\\) is correct
+---------------------------------
+
+Proving that \\(t\_{(j)0}\\) is correct requires first creating commitments to the variables, and then proving a relation over the commitments. For an explanation of how the commitments are created and how the relation is derived, see the [proving that \\(t_0\\) is correct](index.html#proving-that-t_0-is-correct) step of the single-value range proof. The relation over the commitments to prove is:
+
+\\[
+\begin{aligned}
+  t\_{(j)}(x) B + {\tilde{t}}\_{(j)}(x) {\widetilde{B}} \stackrel{?}{=} z^2 z\_{(j)} V_{(j)} + \delta\_{(j)}(y,z) B + x T\_{(j)1} + x^{2} T\_{(j)2}\\\\
+  \delta\_{(j)}(y,z) = (z - z^{2}) \cdot {\langle {\mathbf{1}}, {\mathbf{y}}^{n}\_{(j)} \rangle} - z^{3} z\_{(j)} \cdot {\langle {\mathbf{1}}, {\mathbf{2}}^{n} \rangle}
+\end{aligned}
+\\]
+
+If we combine all of the statements about \\(t\_{(j)0}\\) from all of the `j` parties by adding them together, then we get:
+
+\\[
+\begin{aligned}
+  \sum_{j=0}^{m-1}t_{(j)}(x) B + \sum_{j=0}^{m-1}{\tilde{t}}\_{(j)}(x) {\widetilde{B}} \stackrel{?}{=} z^2 \sum_{j=0}^{m-1} z_{(j)} V_{(j)} + \sum_{j=0}^{m-1} \delta_{(j)}(y,z) B + x \sum_{j=0}^{m-1} T\_{(j)1} + x^{2} \sum_{j=0}^{m-1} T\_{(j)2}
+\end{aligned}
+\\]
+
+We can combine the values and commitments by summing them directly. We can do this instead of having to take a linear combination, because each party's values and commitments are already offset by the values \\(\mathbf{y}^n\_{(j)}\\) and \\(z_{(j)}\\) that are unique to that party.
+
+\\[
+\begin{aligned}
+  t(x) &= \sum_{j=0}^{m-1} t\_{(j)}(x)\\\\
+  {\tilde{t}}(x) &= \sum_{j=0}^{m-1}{\tilde{t}}\_{(j)}(x)\\\\
+  T_1 &= \sum_{j=0}^{m-1} T_{(j)1}\\\\
+  T_2 &= \sum_{j=0}^{m-1} T_{(j)2}\\\\
+  \delta(y,z) &= \sum_{j=0}^{m-1} \delta\_{(j)}(y,z)\\\\
```

Comment:
In the rendered docs this is a little confusing to read, because each of the previous lines are disjoint statements, while this last one has some simplifications. Maybe something like

```
The aggregated values and commitments are
\\[
... stuff ..
\\]
and the expression for \\(\delta(y,z)\\) becomes
\\[
.. delta stuff. ...
\\]
```
so that there's a break in the writing?